lib/combineNodes.js

```diff
@@ -0,0 +1,367 @@
+'use strict';
+var Cesium = require('cesium');
+var jp = require('jsonpath');
+var _ = require('underscore');
```

Comment:
I'm using `_.isEqual` and `_.find` to search for duplicate primitives in an array. That was something else -> optimizing array searching.

I'll take a look around though and see if there's some better choices for that as well.